### ISSUE TITLE
Search & Discovery: Sprint 43 Priorities

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#19695]
## Search & Discovery Team Sprint 43 Priorities
_Aligned with [Quarterly Planning](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/search-discovery/quarterly-planning.md)_
1. **Typeahead 2.0:** Create suggest list, work with search.gov to implement and align with analytics to address fixes
2. **Results Page Redesign:** Design Exploration
3. **Search Collaboration:** Align with UI and other unauthenticated search teams 
4. **XML Sitemap/Structured Markup:** Knowledge transfer from PW team; Define next steps
5. **Click Tracking** (Stretch) Explore best bets implementation
6. **Site Indexing** (Stretch) Exploration

#### *Typeahead POC 2*
- [x] Establish list of typeahead suggestions (PM + Design - #21325) 
List has been established; currently validating results to ensure the suggestions offer user experience we want
- [ ] BUG: Suggestions appearing on two line (FE- #20417) {pulled in from S42- discuss w Sam}
On Hold: Search.gov bug fix removed suggestions that appeared on two lines. Need to determine how many suggestions we have over 32 characters to know if this is still a problem, is fix required
- [x] Analytics fixes for typeahead 0 suggestion states #21648 - related to analytics request #20889 
PR has been merged, analytics should be updated 3/22 with new event logging, testing will occur afterwards

#### *Search Collaboration*
- [ ] Search UI consistency #[21055](https://github.com/department-of-veterans-affairs/va.gov-team/issues/21055) (Megan)
Ongoing issue, will continue into Sprint 44
- [x] Search Collaboration Workshop  #N/A (ALL)
Part 1 complete, part 2 scheduled for 3/26

#### *Results Page Redesign*
- [ ] Design Exploration (Design #21376)
Megan has shared the progress so far which will continue to be iterated on as the UI Search Consistency Discussions continue. As a result this issue will roll into next sprint
- [ ] Right Rail Exploration - waiting on Analytics data request #20168
Expecting analytics to supply GA report link for nav-searchresults link clicks; Denise followed up on 3/22

#### *XML Sitemap/Structured Markup* 
- [x] Transition from PW Team - VA.gov XML Sitemap (PO,PM+FE - #20170) 
Met with PW team, next steps were identified and will be assigned to Sprint 44

#### *Acronym Clarity* #19371
- [ ] Exploration to determine how VA would like to address DB to hold and maintain acronyms (FE - #20428) {Pulled in from S42 to discuss with Sam}
This work has not started yet as Sam was unexpectedly OOO - We plan to evaluate the usage data before we invest in any development exploration. therefore this will be put in the backlog until we have more data. 

## *Stretch Goals*
- [ ] Veteran Touchpoints Summary (PM/All - #18512)
This was a stretch item that has not been prioritized due to lack of time. 

#### *Click Tracking* 
- [ ] Explore Best Bets implementation of Click Tracking (FE Dev- #21514)
Sam 

## *Follow Up/Support  Items*
- [ ] [Analytics] Right Rail Data (#20168)
- [ ] [Analytics] Bounce +Refinement Rate Enhancement- Typeahead Dashboard (#20838)
- [ ] [Analytics] Onsite Search KPI DOMO Dashboard (#16778)

## Sprint Demo
- **No demo for this sprint ** 

## Grooming Discussion Topics
{Team to add as sprint progresses; highlighting issues that need to be created or discussed}_
 - Team did not identify any items to be discussed
